game.py

```diff
@@ -14,13 +14,37 @@
 
 
 def snowman(snowman_word):
-    """Complete the snowman function
-    replace "pass" below with your own code
-    It should print 'Congratulations, you win!'
-    If the player wins and, 
-    'Sorry, you lose! The word was {snowman_word}' if the player loses
-    """
-    pass
+    # The word is provided as a parameter (tests pass it in)
+    snowman_word = snowman_word.lower()
+
+    # Build dict: each unique letter -> False (not guessed yet)
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+
+    # Track wrong guesses as a list
+    wrong_guesses_list = []
+
+    # Loop until too many wrong guesses OR the word is fully guessed
+    while len(wrong_guesses_list) < SNOWMAN_MAX_WRONG_GUESSES and not is_word_guessed(snowman_word, correct_letter_guess_statuses):
```

Comment:
This line of characters exceed the suggested 80 character limit for readability. You'd want to use the [PEP 8 Style Guidelines for line continuation](https://peps.python.org/pep-0008/#indentation) to break this up onto multiple lines. 
```suggestion
    while (
        len(wrong_guesses_list) < SNOWMAN_MAX_WRONG_GUESSES
        and not is_word_guessed(snowman_word, correct_letter_guess_statuses)
    ):
```